guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
 
    .. code-block:: console
    
       $ cp sqream2_config.json sqream3_config.json
       $ vim sqream3_config.json
+      
+**NOTICE:** A unique **instanceID** must be used in each JSON file. IN the example above, the instanceID **sqream_02** is changed to **sqream_03**.
+
+**Comment: The note says sqream_02, but the example says sqream_2, i.e., without the 0. Which is the correct one?**
+
+**Comment: Also, what happens if a non-unique instance ID is used? Depending on the answer, the NOTICE above may have to be a WARNING instead.**
+
+12. **Optional** - If you created additional services in **Step 11**, verify that you have also created their additional configuration files:
+
+    .. code-block:: console
+   
+       $ cp sqream2-service.conf sqream3-service.conf
+       $ vim sqream3-service.conf
+      
+13. For each SQream service configuration file, do the following:
+
+    1. Change the **SERVICE_NAME=sqream2** value to **SERVICE_NAME=sqream3**. **Comment - I think we should show the command line for this step.**
+    
+    2. Change LOGFILE=/var/log/sqream/sqream2.log to LOGFILE=/var/log/sqream/sqream3.log **Comment - I think we should show the command line for this step.**
+    
+14. Set up **servicepicker**:
+
+    1. Do the following:
+
+    .. code-block:: console
+   
+       $ vim /etc/sqream/server_picker.conf
+    
+    2. Change the IP **192.168.5.82** to the value of the **metadataserver IP**:
+    
+    .. code-block:: console
+   
+       $ IP=192.168.5.82 to IP=<metadataserver IP value>
+       
+    3. Change the **CLUSTER** to the value of the cluster path.
+     
+15. Set up your service files:      
+      
+    .. code-block:: console
+   
+       $ cd /usr/local/sqream/service/
+       $ cp sqream2.service sqream3.service
+       $ vim sqream3.service      
+       
+16. Increment each **EnvironmentFile=/etc/sqream/sqream2-service.conf** configuration file for each SQream service file, as shown below:
+
+     .. code-block:: console
+     
+       $ EnvironmentFile=/etc/sqream/sqream3-service.conf
+       
+17. Copy and register your service files into systemd:       
+       
+     .. code-block:: console
+     
+       $ sudo cp metadataserver.service /usr/lib/systemd/system/
+       $ sudo cp serverpicker.service /usr/lib/systemd/system/
+       $ sudo cp sqream*.service /usr/lib/systemd/system/
+       
+18. Verify that your service files have been copied into systemd:
+
+     .. code-block:: console
+     
+       $ ls -l /usr/lib/systemd/system/sqream*
+       $ ls -l /usr/lib/systemd/system/metadataserver.service
+       $ ls -l /usr/lib/systemd/system/serverpicker.service
+       $ sudo systemctl daemon-reload **Comment - Is this a separate step, i.e., to reload the system?**     
+       
+19. Copy the license into the **/etc/license** directory:
+
+     .. code-block:: console
+     
+       $ cp license.enc /etc/sqream/
+
+Configuring an HDFS Environment Under a SQream User
```

Comment:
This entire section should be labeled as optional so it can be skipped if the user does not use HDFS